tests/unit/test_issues_issue.py

```diff
@@ -1,16 +1,29 @@
 """Unit tests for the Issue class."""
 import github3
+import pytest
 
+from github3.issues.label import Label
+from .helper import create_example_data_helper
+from .helper import UnitHelper
+from .helper import create_url_helper
 from . import helper
 
 url_for = helper.create_url_helper(
     'https://api.github.com/repos/octocat/Hello-World/issues/1347'
 )
 
-get_issue_example_data = helper.create_example_data_helper(
+label_url_for = create_url_helper(
```

Comment:
So here we were already importing helper as a module,

We may as well just use that, e.g., 

```py
label_url_for = helper.create_url_helper(
...
)

get_issue_example_data = helper.create_example_data_helper(
...
)
```

I had started slowly moving modules to only importing modules (because it ends up being a lot cleaner) but never finished the whole thing up.